### PR TITLE
generateHTML & generatePDF documentation improvements

### PR DESCRIPTION
**Pull request by JodyWinter**
Not sure what that last line means. Is it supposed to be:

NOTE: For the `generate**Site**` task, there is a different mechanism to change the styles in the generated microsite.

### All Submissions:

* [ ] Did you update the `changelog.adoc`?
* [X] Does your PR affect the documentation?
* [X] If yes, did you update the documentation or create an issue for updating it?

The source of the documentation can be found in `/src/docs/`.

If you didn't find the time to update docs, please create an issue as reminder to do so.

### Your first submission

* [ ] Welcome to the list of contributors! If you have any questions, feel free to ask them by creating a new issue
* [ ] Have you added your name to the list of [contributors.adoc](https://github.com/docToolchain/docToolchain/blob/master/src/docs/manual/05_contributors.adoc)?

inspiration: https://github.com/stevemao/github-issue-templates

### CODE REVIEW

File: src/docs/020_tutorial/030_generateHTML.adoc [hunk 1]
```diff
@@ -120,32 +117,31 @@ The most important ones are
 :pdf-fontsdir: ../pdfTheme/fonts
 ----
 
-For this tutorial, navigate to `src/docs/arc42/arc42.adoc and add
+For this tutorial, navigate to `src/docs/arc42/arc42.adoc then add the following to the top of the document:
```
Comment (jmini) [inline src/docs/020_tutorial/030_generateHTML.adoc]:
```suggestion
For this tutorial, navigate to `src/docs/arc42/arc42.adoc` then add the following to the top of the document:
```

**Comment by jmini** [comment]
@rdmueller IMO this is a good PR and it can be merged.

**Comment by jmini** [comment]
@JodyWinter I would maybe change the title to `generateHTML docs improvements` or something that tells more about what the PR is about.

**Comment by JodyWinter** [comment]
Thank you @jmini :)

**Pull request merged**